wled00/FX.cpp

```diff
@@ -29,7 +34,7 @@
   uint8_t  *fftResult = nullptr;
   float    *fftBin = nullptr;
   um_data_t *um_data;
-  if (usermods.getUMData(&um_data, USERMOD_ID_AUDIOREACTIVE)) {
+  if (UsermodManager::getUMData(&um_data, USERMOD_ID_AUDIOREACTIVE)) {
```

Comment:
I think this is an oversight from @netmindz. AR effects no longer use that approach.

The correct way is: `um_data_t *um_data  = getAudioData();`

Reply:
Yeah possibly missed during one of the rounds on reactoring we did around that. Example perhaps didn't match the actual code exactly so missed from the find and replace